flang/test/Lower/intrinsic-procedures/LEN.f90

```diff
@@ -0,0 +1,32 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
This includes a len_trim test as well. Move that to len_trim test file?

Reply:
My mistake, thanks for pointing out!

Reply:
That's a duplicate, fixed!